### DISCUSSION TITLE
Org-modern not styling todo keywords in log-mode

### DISCUSSION THREAD

**Discussion by kaibagley** [minad/org-modern#225] [Q&A] [answered]
Hey, thanks for the beautiful package!

Emacs: 30.0.50 build 1
Org: 9.8-pre (release_9.7.8-112-gd3d84f)

I just wanted to say that I've noticed that in org agenda log-mode, most of the keywords aren't having the right face applied:

![image](https://github.com/user-attachments/assets/beea9119-e410-4754-8242-eb2218194f5f)

Some do, such as the purple deadline/MEET, there is another day where the deadline has the correct face, but none of the Clocked or Closed lines have it. The screenshot is from `M-x org-agenda` > Agenda for current week or day > `M-x org-agenda-log-mode`.

Relevant init.el:
```
;; Org modern
(use-package org-modern
  :config
  (setq org-modern-block-fringe nil
        org-modern-priority-faces '((?A :background "#291f26" :foreground "#b52c2c")
                                    (?B :background "#291f26" :foreground "#b08940")
                                    (?C :background "#291f26" :foreground "#bf9f8f"))
        org-modern-todo-faces '(("NEXT" :background "midnight blue" :foreground "dodger blue")
                                ("WAIT" :background "#4f2f00" :foreground "orange")
                                ("INTR" :background "#a94a91" :foreground "#fdb4eb")))
  (if (display-graphic-p)
      (setq org-modern-table t)
    (setq org-modern-table nil))
  (global-org-modern-mode))
```

I'm unsure of when it happened exactly, but I only noticed today after I ran `straight-pull-all`, which was presumable when I upgraded org-modern to version 1.4 (which it is according to `pkg-info`)

Thanks a lot!

**Comment by minad** [comment, marked as answer]
There have been some changes regarding the agenda highlighting recently. I don't observe such problems with Org 9.8 and Emacs 30. Which Emacs/Org version do you use? Could you upgrade Org to 9.7.8?

EDIT: I checked the clocking functionality and I could reproduce the problem. Fix in https://github.com/minad/org-modern/commit/b9fce9786f8433d20212c72ec57757593a5ac81c.

**Reply by kaibagley** [reply]
Sweet thanks for checking that out and swiftly fixing it!! Will mark this as closed when I can install the update and check it out

`org-version` 9.8-pre (release_9.7.8-112-gd3d84f)

**Reply by kaibagley** [reply]
Sorry for late response, but I can confirm it is fixed for me. 

Thanks for your continued hard work, you're amazing!
